crates/build/src/crate_metadata.rs

```diff
@@ -137,6 +137,13 @@ impl CrateMetadata {
     pub fn metadata_path(&self) -> PathBuf {
         self.target_directory.join(METADATA_FILE)
     }
+
+    /// Get the path of the contract bundle, containing metadata + code
```

Comment:
```suggestion
    /// Get the path of the contract bundle, containing metadata + code.
```

Just for consistency with the others.